pkgs/top-level/python-packages.nix

```diff
@@ -8751,6 +8751,8 @@ in {
     };
   };
 
+  pepper = callPackage ../development/python-modules/pepper { };
```

Comment:
@PierreR I'm fine with any really, I don't know what would be best there. Anyway, ansible is an unfortunate case.